setup-readme.sh

```diff
@@ -6,13 +6,21 @@ user="$1"
 repo="$2"
 shift 2
 
+default_branch=$(git rev-parse --abbrev-ref origin/HEAD 2>/dev/null | sed 's,.*/,,')
+if [ "$default_branch" = "HEAD" ]; then
+    # Do this conditionally so that users can manually configure it.
+    git remote set-head -a origin
```

Comment:
I think that perhaps it is time to make a utility for this.  I say that because the code in `id.mk` (which runs before this does) won't do `git remote set-head -a` and this won't respect `GIT_REMOTE` from the makefile.